workflow/state-machines.rst

```diff
@@ -209,6 +209,33 @@ you can get this state machine by injecting the Workflow registry service::
         public function someMethod($subject)
         {
             $stateMachine = $this->workflows->get($subject, 'pull_request');
+            $stateMachine->apply($subject, 'wait_for_review');
+            // ...
+        }
+
+        // ...
+    }
+
+You can also directly inject the proper :class:`Symfony\\Component\\Workflow\\StateMachine`
```

Comment:
Do it works out of the box ?
I will work only if you have one and only one StateMachine?
In my project, I had to do that to make it work:

```yaml
services:
    # We have only one state machine, so we can be lazy
    Symfony\Component\Workflow\StateMachine:
        alias: state_machine.order
```

Reply:
Are you speaking of autowiring?
Actually in many of my projects i am still on 3.4 lts without using autowire, so when i wrote the StateMachine i also wrote the service id

but yes in case of autowiring actuivated, with ≥ 2 StateMachine, we need to create aliases

I do not know how to properly document such different use case depending on the code